test/orange/data/rect_array.org.omn

```diff
@@ -0,0 +1,117 @@
+!##############################################################################
+! File  : Geometria/orange/test/data/array.org.omn
+!
+! Note that this is now different from the GG geometry: the array has two
+! elements along Z.
```

Comment:
Replace this header with the one from `five-volumes.org.omn` and delete the footer. Consider using the thinner separators `!~~~~!` instead of the thick hashes.